src/main/resources/phrases_en_US.properties

```diff
@@ -32,6 +32,7 @@ QnMerge.start=OK, I'll try to merge now. You can check the progress of the merge
 QnMerge.already-closed=The pull request is closed already, so I can't merge it
 QnMerge.head-is-gone=Head repository is gone, can't merge from it
 QnMerge.base-is-gone=Base repository is gone, can't merge into it
+QnMerge.checks-are-failed=Can't merge it, some checks were failed.
```

Comment:
@volodya-lombrozo I would make a longer text here, with more details. Some people might be concerned about this situation and will think that the problem is in Rultor